targeteditor/src/main/scala/explore/targeteditor/RVInput.scala

```diff
@@ -87,29 +98,21 @@ object RVInput {
             FormInputEV(
               id = state.rvView.tag,
               label = state.rvView.tag,
-              value = props.value.zoom(unsafeRVtoZLens),
+              value = props.value.zoom(rvToRedshiftGet)(rvToRedshiftMod),
```

Comment:
In cases where we don't have a `Lens` available, it's OK to zoom via get/mod functions.

Also, we allow setting to be done via `InputFormEV` calling the `View`'s `set` rather than doing it manually `onBlur`.

Reply:
I like this, I was thinking to do it as the lens are unlawful.